core/src/main/resources/filodb-defaults.conf

```diff
@@ -25,7 +25,7 @@ filodb {
   }
 
   # # of shards each application/metric is spread out to = 2^spread
-  default-spread = 1
+  spread.default = 1
```

Comment:
Please add a large block comment explaining the format of the spread {... } with an example.  The defaults file should contain very very verbose explanations of each option.

Reply:
Added description